Editor/Helper/AddressablesImporterOdinHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace UnityAddressableImporter.Editor.Helper
+{
+    using System;
+    using Object = UnityEngine.Object;
+
+#if ODIN_INSPECTOR
+    
+    using Sirenix.OdinInspector.Editor;
+    using Sirenix.OdinInspector;
+    
+    public class AddressablesImporterOdinHandler : IDisposable
+    {
+        private Object _target;
+        private PropertyTree _drawer;
+        
+        public void Initialize(Object target)
+        {
+            _target = target;
+            _drawer = PropertyTree.Create(_target);
+        } 
+        
+        public void Draw() => _drawer?.Draw(false);
+
+        public void Dispose()
+        {
+            _target = null;
+            _drawer?.Dispose();
+            _drawer = null;
+        }
+    }
+    
+#else
+
+    public class AddressablesImporterOdinHandler : IDisposable
```

Comment:
Also, rename here.

Reply:
What name should be?

Reply:
AddressablesImporterOdinHandler => AddressableImporterOdinHandler

I explained before that it was a mistake (from me) to name class using "AddressableImporter" (singular), includes the package name. The Unity team uses "Addressables" (plural). So to avoid more confusion, I just keep every "AddressableImporter" singular.

Reply:
+